### PR TITLE
LRDOCS-2232 Applied Liferay styles to API docs

### PR DESCRIPTION
**Pull request by mwilliams2014**
@jhinkey updated styles. Please test them locally and let me know what you think.

### CODE REVIEW

**Comment by jhinkey** [comment]
Thanks, Mike! Reviewing now.

**Comment by mwilliams2014** [comment]
Updated styles for ce pushed.

**Comment by mwilliams2014** [comment]
@jhinkey updated structure for styles.

File: tools/styles/ce/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* Javadoc style sheet */
```
Comment (jhinkey) [inline tools/styles/ce/stylesheet.css]: To make naming consistent, this should be named after the reference doc it styles (i.e., javadoc.css).

File: styles/DTDDocStyle.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+body { background-color:white; font-family: Verdana, Geneva, Arial, Helvetica, sans-serif; font-size: small; }
```
Comment (jhinkey) [inline styles/DTDDocStyle.css]: All the CSS files belong under tools/styles/${lp.style.edition} either directly or in a subdirectory, as in the case of taglibs/stylesheet.css (which needs to keep that specific name).

And the naming convention is not consistent with Liferay's CSS file name convention. Please refer to existing CSS files.

**Comment by mwilliams2014** [comment]
@jhinkey Ready for review

**Comment by mwilliams2014** [comment]
@jhinkey CSS is formatted. I did not find the trailing whitespace Brian was talking about in the build.xml.

**Comment by mwilliams2014** [comment]
@jhinkey updated :-)

**Comment by jhinkey** [comment]
Started reviewing. Thanks, Mike.

**Comment by mwilliams2014** [comment]
@jhinkey SF fixed :-)

**Comment by mwilliams2014** [comment]
@jhinkey updated

**Comment by jhinkey** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/39614

**Pull request closed without merging**